Packs/AWS-Organizations/Integrations/AWSOrganizations/README.md

```diff
@@ -0,0 +1,1119 @@
+#### Context Example
+```json
+{
+    "AWS": {
+        "Organizations": {
+            "Tag": [
+                {
+                    "Key": "test",
+                    "ResourceId": "ou-ab12-abcd1234",
+                    "Value": "true"
+                },
+                {
+                    "Key": "new",
+                    "ResourceId": "ou-ab12-abcd1234",
+                    "Value": "true"
+                },
+                {
+                    "Key": "key",
+                    "ResourceId": "ou-ab12-abcd1234",
+                    "Value": "value"
+                }
+            ],
+            "TagNextToken": null
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### AWS Organization *ou-ab12-abcd1234* Tags
+>|Key|Value|
+>|---|---|
+>| test | true |
+>| new | true |
+>| key | value |
+
+### aws-org-account-create
+
+***
+Creates an AWS Account that is automatically a member of the organization.
+
+#### Base Command
+
+`aws-org-account-create`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| account_name | The friendly name of the member account. | Required | 
+| email | The email address of the owner to assign to the new member account. This email address must not already be associated with another Amazon Web Services account. Use a valid email address to complete account creation. | Required | 
+| iam_user_access_to_billing | If set to ALLOW, the new account enables IAM users to access account billing information if they have the required permissions. If set to DENY, only the root user of the new account can access account billing information. Possible values are: Allow, Deny. Default is Allow. | Optional | 
+| role_name | The name of an IAM role that AWS Organizations automatically pre-configures in the new member account. This role trusts the management account, allowing users in the management account to assume the role, as permitted by the management account administrator. The role has administrator permissions in the new member account. Default is OrganizationAccountAccessRole. | Optional | 
+| tags | A comma-separated list of tags to attach to the newly created account. Each tag should be in the format: "key=value". | Required | 
+| request_id | The Id of the create request that is used for polling. | Optional | 
+| interval_in_seconds | Indicates how long to wait between command executions (in seconds) when the 'polling' argument is true. Minimum value is 10 seconds. Default is 30. | Optional | 
+| timeout | Indicates the time in seconds until the polling sequence times out. Default is 600. | Optional | 
+| roleArn | The Amazon Resource Name (ARN) of the role to assume. | Optional | 
+| roleSessionName | An identifier for the assumed role session. | Optional | 
+| roleSessionDuration | The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role. | Optional | 
+
```

Comment:
In the YML file there is an argument called hide_polling_output (line 333).  This argument does not appear here in the README.

Reply:
it's  a hidden argument